.travis.yml

```diff
@@ -1,94 +0,0 @@
-version: ~> 1.0
-dist: focal
-# this arch is required as is for Partner Queue Solution - DO NOT MODIFY
-arch: ppc64le
-language: java
-
-cache:
-  directories:
-    - ~/.m2
-    - ~/.ivy2
-
-branches:
-  only:
-    - master
-
-before_install:
-  - sudo apt update
-
-install:
-  - sudo apt install -y groovy xsltproc xmlstarlet ant
-  - ./.ci/travis.sh install-adoptium-jdk
-  - ./.ci/travis.sh install-custom-mvn
-
-jobs:
-  fast_finish: true
-  include:
-
-    # this job do deploy maven repository
-    # unit tests (openjdk11)
-    - jdk: openjdk11
-      env:
-        - DESC="tests and deploy"
-        - CMD="mvn -e --no-transfer-progress clean integration-test failsafe:verify
-          -DargLine='-Xms1024m -Xmx2048m'"
-        - DEPLOY="true"
-        - USE_MAVEN_REPO="true"
-
-    # sevntu (openjdk11)
-    - jdk: openjdk11
-      env:
-        - DESC="sevntu"
-        - CMD="mvn -e --no-transfer-progress compile antrun:run@ant-phase-verify-sevntu -Psevntu"
-        - DEPLOY="true"
-        - USE_MAVEN_REPO="true"
-
-    # codecov
-    - jdk: openjdk11
-      env:
-        - DESC="codecov"
-        - USE_MAVEN_REPO="true"
-        - CMD="./.ci/travis.sh jacoco"
-
-    - jdk: openjdk11
-      env:
-        - DESC="codenarc analysis"
-        - CMD="./.ci/codenarc.sh"
-
-    - jdk: openjdk11
-      env:
-        - DESC="no-exception-samples-ant"
-        - CMD="./.ci/no-exception-test.sh no-exception-samples-ant"
-
-    # Ensure that all modules are used in no exception configs
-    - env:
-        - DESC="ensure that all modules are used in no exception configs"
-        - CMD1="export PULL_REQUEST=$TRAVIS_PULL_REQUEST"
-        - CMD2="./.ci/validation.sh verify-no-exception-configs"
-        - CMD="$CMD1 && $CMD2"
-
-before_script:
-  - |
-    if [[ $TRAVIS_CPU_ARCH == 'ppc64le' ]]; then
-        export JAVA_HOME=/usr/lib/jvm/adoptopenjdk-11-hotspot-ppc64el
-        export PATH=$JAVA_HOME/bin:$PATH
-    fi
-  - groovy --version
-
-script:
-  # manually set JAVA_HOME to overcome issue with travis ci noted at
-  # https://github.com/checkstyle/checkstyle/pull/11699#issue-1261272652
-  # - JAVA_HOME='/usr/lib/jvm/adoptopenjdk-11-hotspot-ppc64el'
-  - ./.ci/travis.sh init-m2-repo
-  - ./.ci/travis.sh run-command "$CMD"
-  # - ./.ci/travis.sh remove-custom-mvn
-  # - ./.ci/travis.sh remove-adoptium-jdk
-  - ./.ci/validation.sh git-diff
```

Comment:
More just a question tagging onto @romani 's, should we be ensure all these other CIs run a git-diff check?

Reply:
I will add this to all

Reply:
We have this in a bunch of CI:
```
➜  checkstyle git:(move-travis-tasks) ✗ ag -Q "./.ci/validation.sh git-diff" --hidden
.circleci/config.yml
53:          command: ./.ci/validation.sh git-diff

.semaphore/semaphore.yml
64:            - ./.ci/validation.sh git-diff
86:            - ./.ci/validation.sh git-diff
105:            - ./.ci/validation.sh git-diff

azure-pipelines.yml
167:      ./.ci/validation.sh git-diff

.travis.yml
54:  - ./.ci/validation.sh git-diff


```

And it is already in semaphore, so "done"